gemini/prompts/prompt_optimizer/get_started_with_vertex_ai_prompt_optimizer_sdk.ipynb

```diff
@@ -0,0 +1,1161 @@
+{
```

Comment:
<p>And you if you &gt; And you</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/1154/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
done